### ISSUE TITLE
Genders other than Male and Female

### ISSUE THREAD

**Issue by Youhan** [schemaorg/schemaorg#2898]
How do you specify genders other than Male and Female in Person schema?

**Comment by WeaverStever** [comment]
I would say that you use the GenderType enumeration and provide a url that fits your subject.
https://schema.org/GenderType

A place to find urls might be here:
https://en.wikipedia.org/wiki/Category:Gender

The ISO standards are here
https://en.wikipedia.org/wiki/ISO/IEC_5218

**Comment by danbri** [comment]
We have tried explicitly to be flexible, respectful and inclusive. The two properties that reference gender are https://schema.org/gender and https://schema.org/suggestedGender for people-audience, and they both lean heavily towards free-text. While there is an enumeration with just Male and Female in, the intent was for those to be usable alongside other textual values. The Enumeration mechanism doesn't explicitly allow for non-enumerated options to be listed in the enumeration but we could make that more explicit in the text. Those definitions do mention a couple of other text-strings that might be used - 'unisex' and 'mixed' - both of which are more about non-human things that get gender categories e.g. Products (typically garments), sports teams etc.

**Comment by nedjo** [comment]
@danbri 

> We have tried explicitly to be flexible, respectful and inclusive. The two properties that reference gender are https://schema.org/gender and https://schema.org/suggestedGender for people-audience, and they both lean heavily towards free-text

I'm coming from the Drupal project, specifically the [Schema.org Blueprints module](https://www.drupal.org/project/schemadotorg), where an ambitious initiative is underway to programmatically derive content models and their properties from schema.org. One of the issues that's come up is the gender property. This attempt to implement the schema has produced a user profile gender field with two options: Male and Female. We could use some guidance here on how best to implement the spec--more on that below.

Overall it looks like, despite the stated intent, #541 has led to erosion of the original designation of gender as free-text.

Which, let's be frank, is hardly a shocker. The introduction of a controlled vocabulary for gender in which "Male" and "Female" are the only recognized values was never going to be an unambiguous coup for inclusiveness. See for example https://github.com/kibimon/spec/issues/17.

At a minimum, to avoid resonating with and potentially reinforcing a regressive status quo, `GenderType` would presumably need to be expanded to include options that don't align cognitively with the cisgender binary. Just as governments are slowly, slowly introducing somewhat more inclusive taxonomies.

Potentially relevant sources:

* Statistics Canada
    * [Classification of gender](https://www23.statcan.gc.ca/imdb/p3VD.pl?Function=getVD&TVD=1326727&CVD=1326727&CLV=0&MLV=1&D=1), which recognizes a "Non-binary person" in addition to "Man" and "Woman"
    * [Classification of cisgender, transgender and non-binary](https://www23.statcan.gc.ca/imdb/p3VD.pl?Function=getVD&TVD=1326715&CVD=1326716&CLV=0&MLV=2&D=1)
* US CDC HIV prevention and care guidelines on [Collecting Sexual Orientation and Gender Identity Information](https://www.cdc.gov/hiv/clinicians/transforming-health/health-care-providers/collecting-sexual-orientation.html), which lists five categories for gender identity in addition to a free text field or "decline to answer".
* The UK 2021 census, which listed the question "Is the gender you identify with the same as your sex registered at birth?" and a free-text field for specifying a gender identity, see [this BBC article](https://www.bbc.com/news/uk-wales-55721123).

Back to the Drupal case.

Per Drupal core, we have two potentially applicable combinations of data storage and UI: (a) a free-text field and (b) an enumeration/radios (options: "Male" or "Female"). Which of these two best fits the schema.org `gender` property? And - critically for this purpose - can the best choice be derived from the schema itself?

The thing is, the "Male/Female" gender select didn't come from ignoring the schema.org spec--it came from trying to faithfully implement it.

Is free-text always the more appropriate model for a property where there is both `Text` and `SomeType`?

[Gender](https://schema.org/gender) has two values for `rangeIncludes`, `Text` and `GenderType`. So schema-wise it's parallel to something like https://schema.org/paymentStatus, which similarly accepts `Text` or `PaymentStatusType`.

Does `PaymentStatus` also "lean heavily towards free-text"? That seems possible, but not particularly likely.

Is the _order_ of `rangeIncludes` values significant? Can/should an implementation assume free text is primary if `Text` is listed _before_ a specific type such as `PaymentStatusType`?

What seems to distinguish `gender` from `paymentStatus` above all would be, um, consciousness or agency. A `Person` in most cases has consciousness and can say for themself what their gender is. An `Invoice` doesn't, and can't. Not yet anyway ;) So, when `rangeIncludes` is a combination of `Text` and `SomeType`, is there a way to infer appropriate structure + UI from the type a property is attached to?

Or is the "lean" to be inferred only from the property `comment` (description), and implemented through special-casing of specific properties?

The Drupal.org issue is [Gender incorrectly limited to enumeration](https://www.drupal.org/project/schemadotorg/issues/3280561). Guidance or feedback either here or there would be most welcome.

**Comment by nedjo** [comment]
> At a minimum, to avoid resonating with and potentially reinforcing a regressive status quo, `GenderType` would presumably need to be expanded to include options that don't align cognitively with the cisgender binary.

To clarify, I'm really just flagging an issue, not suggesting a particular way forward to addressing it. Any effort to codify and make machine readable an attribute like gender is going to hit a bunch of ethical, privacy, cultural, and perhaps even legal complexities. There are lots of good resources out there that could provide guidance. The [Gender Field](https://www.drupal.org/project/gender) project has some thoughtful suggestions as well as links to relevant resources. Expanding the terms in `GenderType` is one possible direction, though it's boxed in by the decision to apply the gender property to `SportsTeam` and `SizeSpecification` as well as `Person`. There are only so many descriptors of any sort that apply equally well to a person, a sports team, and a product. We're already seeing this in description text that suggests the term `Mixed` for application to `SportsTeam` but not to `Person`.

In addition to other issues, the current `Male`/`Female` schema:

* Risks conflation of gender with biological sex. Other schema may partially mitigate this risk by using, for example, `Man` and `Woman`.
* Is not explicit about how inclusive or otherwise those terms are. Are they encompassing of cisgender, transgender, and other identities? A [descriptor like](https://schema.org/Male) "The male gender" doesn't indicate.

Besides expanding/improving `GenderType`, other options might include:

* Create one or more new enumerations that are explicitly about people and don't conflate the gender of a person with the gender of a thing.
* Divide gender identity into multiple properties, as Statistics Canada did to avoid conflating a `Female`/`Male`/`Non-binary` distinction with `Cisgender`/`Transgender`/`Non-binary`.
* Just back out. Remove `GenderType` and leave the gender property to its free-text origins.

And no doubt many other thoughtful and creative approaches. Whatever the approach, to belabour the obvious, presumably it would best be entered into with involvement and leadership from a diversity of relevant perspectives and experience.

And on that note, I'll sign out, having doubtless said plenty already ;)

**Comment by Metonical** [comment]
1. I am really glad this discussion is still open and not been closed! I would want to see the adding of Non binary and prefer not to say options just like Canada and America use. Using string data as suggested by @danbri  from the other option box  would mean this data would not be as easily processed  in comparison to the male/female option risking effecting how the data would be processed and presented. Providing the further non binary and prefer not to say options would increase the strength of the data collected and provide a better end user experience at the same time. By only having the two options you are limiting the system, decreasing the usefulness of the data and diminishing the end user experience all at the same time.

**Comment by aredridel** [comment]
I definitely can't encode my legal gender here.

**Comment by bk2204** [comment]
I also have problems encoding my legal gender here.

I think there's a great deal of nuance in gender and simply offering a third option (say, “other” or “non-binary”) is not going to be satisfactory.  Gender, like sexuality or ethnicity, is very subtle and the language used is extremely personal and is highly influenced by culture, especially given that some cultures have acknowledged specifically named third (or fourth, etc.) genders for hundreds of years.  Many of those people in those other genders might not consider themselves as non-binary per se.

For context, I'll link to [the WikiData page for “sex or gender”](https://www.wikidata.org/wiki/Property:P21) just to demonstrate the variety of options, plus the fact that they still got it wrong with a one-of constraint.  Most of the people I have known who are gender diverse would fit into multiple categories in that list.

Also, asking about gender is extremely fraught in general because it depends on the context.  Are you asking me what the entry is on my passport because I'm flying somewhere?  If so, what happens if I have multiple and they disagree?  Do you just need something on _an_ ID?  Do you need what's on my provincial health record?  Are you just trying to address me in a respectful way (which knowing my gender doesn't tell you how to do)?  Are you inquiring how I describe my gender for human subject research purposes?  Are you asking how I view myself notwithstanding my government documents?  What if I'm not sure or questioning?  What if I'm uncomfortable providing information to the requesting party?  The answer to lots of these might differ for a lot of folks who are in various stages of transition or who have relationships with locales or governments where they may not have full legal recognition.  This is why I recommend not asking about gender on most forms.

Finally, I want to point out that the use of the schema as it stands almost certainly poses legal problems.  Both Ontario (at the provincial level) and Canada (at the federal level) outlaw discrimination on the basis of gender identity or expression.  The [Canadian Human Rights Commission gives this example](https://www.chrc-ccdp.gc.ca/individuals/human-rights/about-discrimination):

> A policy requires that a person identifies themselves as either male or female. This may be a case of discrimination based on gender identity or expression

Even privileging certain defined options may be legally problematic and it certainly will leave a bad taste in many folks' mouths.  I'd strongly suggest removing GenderType as an enumeration altogether and issuing an update deprecating the old versions.

**Comment by aredridel** [comment]
Yeah. Not sure that a full free form fill-in is actually useful ever, but at least ICAO-compatible options are a start.

That said there are legal concepts to encode too. The question is which ones.
